### ISSUE TITLE
Outdated version string in __init__.py

### ISSUE THREAD

**Issue by garoto** [ihabunek/twitch-dl#29]
```
> pip install twitch-dl==1.10.1
Requirement already satisfied: twitch-dl==1.10.1 in j:\misc\python-3.8.2-amd64\lib\site-packages (1.10.1)
Requirement already satisfied: m3u8<0.4,>=0.3.12 in j:\misc\python-3.8.2-amd64\lib\site-packages (from twitch-dl==1.10.1) (0.3.12)
Requirement already satisfied: requests<3.0,>=2.13 in j:\misc\python-3.8.2-amd64\lib\site-packages (from twitch-dl==1.10.1) (2.23.0)
Requirement already satisfied: iso8601 in j:\misc\python-3.8.2-amd64\lib\site-packages (from m3u8<0.4,>=0.3.12->twitch-dl==1.10.1) (0.1.12)
Requirement already satisfied: idna<3,>=2.5 in j:\misc\python-3.8.2-amd64\lib\site-packages (from requests<3.0,>=2.13->twitch-dl==1.10.1) (2.9)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in j:\misc\python-3.8.2-amd64\lib\site-packages (from requests<3.0,>=2.13->twitch-dl==1.10.1) (1.25.9)
Requirement already satisfied: chardet<4,>=3.0.2 in j:\misc\python-3.8.2-amd64\lib\site-packages (from requests<3.0,>=2.13->twitch-dl==1.10.1) (3.0.4)Requirement already satisfied: certifi>=2017.4.17 in j:\misc\python-3.8.2-amd64\lib\site-packages (from requests<3.0,>=2.13->twitch-dl==1.10.1) (2020.6.20)

[ J:\Misc\python-3.8.2-amd64\Lib\site-packages\twitchdl ]

> twitch-dl.exe --version
twitch-dl v1.9.0

[ J:\Misc\python-3.8.2-amd64\Lib\site-packages\twitchdl ]

> cat __init__.py
__version__ = "1.9.0"

CLIENT_ID = "kimne78kx3ncx6brgo4mv6wki5h1ko"
```

**Comment by ihabunek** [comment]
Thanks. That's what happens when you don't have an automated release procedure. :)

**Comment by garoto** [comment]
Thank you. Was worried it could be a Python+Windows specific snafu somewhere.

**Comment by ihabunek** [comment]
Fixed in [1.10.2](https://github.com/ihabunek/twitch-dl/releases/tag/1.10.2)
